src/Atmos/Model/moisture.jl

```diff
@@ -239,3 +240,128 @@ function flux_diffusive!(moist::EquilMoist, flux::Grad, state::Vars, d_q_tot)
     flux.ρu += d_q_tot .* state.ρu'
     flux.moisture.ρq_tot += d_q_tot * state.ρ
 end
+"""
+  NonEquilMoist
+
+  Assumes that there isn't any thermodynamic equilibrium
+  
+  Adds addtional moisture variable for cloud water ρq_liq and cloud ice ρq_ice::FT that are transported in the same way as ρq_tot
+
+  The equilibrium phase partition is calculated and used as a reference to compute the source term for the cloud phase.
+
+"""
+
+struct NonEquilMoist <: MoistureModel end
+
+vars_state(::NonEquilMoist, FT) = @vars(ρq_tot::FT, ρq_liq::FT, ρq_ice::FT)
+vars_gradient(::NonEquilMoist, FT) =
+    @vars(q_tot::FT, q_liq::FT, q_ice::FT, h_tot::FT)
+vars_diffusive(::NonEquilMoist, FT) = @vars(
+    ∇q_tot::SVector{3, FT},
+    ∇q_liq::SVector{3, FT},
+    ∇q_ice::SVector{3, FT}
+)
+vars_aux(::NonEquilMoist, FT) =
+    @vars(temperature::FT, θ_v::FT, q_liq::FT, q_ice::FT, p::FT, src_qliq::FT)
```

Comment:
```suggestion
vars_state(::NonEquilMoist, ::Auxiliary, FT) =
    @vars(src_qliq::FT)
```
We really shouldn't be adding more fields to the aux state.